infrastructure/src/main/java/org/corfudb/infrastructure/ServerHandshakeHandler.java

```diff
@@ -107,7 +107,7 @@ public void channelRead(ChannelHandlerContext ctx, Object m) throws Exception {
         // Validate handshake, but first verify if node identifier is set to default (all 0's)
         // which indicates node id matching is not required.
         if (serverId.equals(UUID.fromString("00000000-0000-0000-0000-000000000000"))) {
-            log.info("channelRead: node id matching is not requested by client.");
+            log.trace("channelRead: node id matching is not requested by client.");
```

Comment:
Do we see this log message often?
If not, there is no point in reducing the level.

Reply:
All log levels I changed I see pretty often in the log. The number of those messages growing pretty fast in any connection issue happens, there are many duplicating messages, the log level some of them can be lowered. There are 3 classes which generate most of  the messages in a log: ServerHadshakeHandler, NettyClientRouter, NettyServerRouter

Reply:
Also since we don't use `serverId` I don't see any reasons to print this message